linter.py

```diff
@@ -71,24 +111,50 @@ def split_match(self, match):
 
         if match:
             open_file_name = os.path.basename(self.view.file_name())
-            linted_file_name = match.group('file_name')
+            # Since the filename on the top row might be different than the open file if, for example,
+            # something is imported from another file. Use the filename from the footer is it's available.
+            linted_file_name = match.group('file_name_2') or match.group('file_name_1')
```

Comment:
The file name from the "footer" isn't where the error is unless its the same file name from the first line so this isn't needed.